x/superfluid/keeper/stake.go

```diff
@@ -311,6 +314,7 @@ func (k Keeper) SuperfluidUndelegateToConcentratedPosition(ctx sdk.Context, send
 // partialUndelegateCommon acts similarly to undelegateCommon, but undelegates a partial amount of the lock's delegation rather than the full amount. The amount
 // that is undelegated is placed in a new lock. This function returns the intermediary account associated with the original lock ID as well as the new lock that was created.
 // An error is returned if the amount to undelegate is greater than the locked amount.
+// nolint: unused
 func (k Keeper) partialUndelegateCommon(ctx sdk.Context, sender string, lockID uint64, amountToUndelegate sdk.Coin) (intermediaryAcc types.SuperfluidIntermediaryAccount, newlock *lockuptypes.PeriodLock, err error) {
```

Comment:
Should we remove this? I'm generally for removing any unused or poorly tested code.

If this is needed in the future, the author can review git history if needed. At the same time, it would grant the opportunity to re-implement the method with fresh invariants in mind.

Reply:
Addressed in https://github.com/osmosis-labs/osmosis/pull/5949/commits/96f486b5e1bbf468406fc4b5d1aa3e17ef6b4003